### ISSUE TITLE
VASL 6.2 release notes

### ISSUE THREAD

**Issue by davidsullivan317** [vasl-developers/vasl#129]
Create release notes for VASL 6.2 on vasl.info.

**Comment by davidsullivan317** [comment]
Hey Sam, Kedge - 6.2 is ready to go. Can you please finalize the release notes and make it available for download on vasl.info? I flushed out the release notes a bit here: 

Let's make sure the NOTICE is front and center so folks don't miss it. 

If you need me to build the module let me know. 

Kedge - can you send me your full name so I can thank you for your work on the documentation in the release message?

**Comment by davidsullivan317** [comment]
Sorry - missed the link. http://forums.gamesquad.com/showthread.php?120802-VASL-6-2-is-available-for-testing

**Comment by wintrymix** [comment]
I've added the additional information you posted to GS.  I added a Bold, Italicized "Important Note" to the list item about the expanded pixels.  Have a quick review to make sure you think it stands out enough.  

Outside the ASL community, I go by Bob Johnson, but most in the ASL community know me as Kedge.  When I write for Dispatches from the Bunker, they usually credit me as "Bob 'Kedge' Johnson".
